components/blocks/gridLayout.tsx

```diff
@@ -23,13 +33,32 @@ export const GridLayout = ({ data }: GridLayoutProps) => {
     <>
       {data.grids?.map((grid, i) => (
         <Container padding="pt-0" key={i}>
-          <h2 className="text-xl text-ssw-red">{grid.gridTitle}</h2>
+          {grid.showGridTitle && (
+            <h2
+              data-tina-field={tinaField(grid, "gridTitle")}
+              className={`${grid.centeredGridTitle ? "text-center text-2xl font-light" : "text-xl text-ssw-red"} my-3`}
```

Comment:
Up to you, but I think it looks a little thin compared to the main heading
```suggestion
              className={`${grid.centeredGridTitle ? "text-center text-2xl" : "text-xl text-ssw-red"} my-3`}
```

Reply:
Also the colour maybe shouldn't be depndent on the centre value, maybe make it a different type of heading if you want (h2s are always red for prose headings in `tailwind.config.js`)

Reply:
agree. i removed the font-light class.

As we discussed, leaving the red color based on conditional